apps/condo/domains/ticket/hooks/useTicketExportTask.tsx

```diff
@@ -16,6 +16,17 @@ export const useTicketExportTaskUIInterface = () => {
     const TicketExportTaskProgressDescriptionCompleted = intl.formatMessage({ id: 'tasks.TicketExportTask.progress.description.completed' })
     const router = useRouter()
 
+    const tryToDownloadFile = (taskRecord: TicketExportTask) => {
+        if (window) {
+            if (taskRecord.file) {
+                console.log('Downloading exported file TicketExportTask', taskRecord)
```

Comment:
useless log, remove please

Reply:
We are aggregating logs with Sentry, it will help us with debugging. Also we can integrate Amplitude here